shell/platform/darwin/macos/framework/Source/FlutterView.h

```diff
@@ -4,35 +4,40 @@
 
 #import <Cocoa/Cocoa.h>
 
-/**
- * Listener for view resizing.
- */
-@protocol FlutterViewReshapeListener <NSObject>
+@protocol FlutterViewDelegate <NSObject>
```

Comment:
Can we call this `FlutterSurfaceDelegate` and move `present` and `getFrameBufferIdForWidth` to this?